### ISSUE TITLE
No recently added anything

### ISSUE THREAD

**Issue by Rudd-O** [jtbgroup/kodi-media-sensors#28]
**Describe the bug**

The recently added TV sensor shows as off, even though I can see in Kodi that there's new TV shows.  Also, the attributes show this!

```
Data
[{"title_default": "$title", "line1_default": "$episode", "line2_default": "$release", "line3_default": "$rating - $runtime", "line4_default": "$number", "icon": "mdi:eye-off"}] 
```

This is weird.  It was working before.

This naturally causes the card not to show anything.

**Home Assistant and Custom Component versions (please complete the following information):**

Home Assistant version: 2023-04-01

kodi-recently-added component version: 5.1.1

**Comment by LukeBH** [comment]
I have the same issue, in addition the sensors all show with names like kms_m_dec0746d3633e52946e6fbc6c69508ff meaning that I can't identify between them.

**Comment by jtbgroup** [comment]
Hi, 

I just published a version 5.1.2 that (should) fix the issue related to the "recently added" stuff.

@LukeBH ,
There was a breaking change with version 5.x. To allow multiple instances of the sensors, I had to attribute a unique ID to every sensor. By default, the entity id is the same as the unique id, but you can easily change that manually in HA. I'll try to have something more human readable in a future release.

@Rudd-O ,
If everything ok for you with the version 5.1.2, can you close the ticket please?

**Comment by Rudd-O** [comment]
You can close the ticket.  It is fixed now.
